UKHomeOffice/cerberus-service
# RoRo-V2 - Task list page

## Description
This PR addresses a bug where the roro-v2 page would block users from seeing tasks even though roro tasks are not pnr related.

## To Test
- Pull & run against dev
- Take the unhappy path in the initial PNR form
- Access the RoRo V2 task list page and tasks should load up.
- Access Airpax task list page and the user should be blocked from seeing any tasks.

## Developer Checklist

\* Required
